applications/server/controllers/synchronization.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+import json
+
+def index():
+    """
+    example action using the internationalization operator T and flash
+    rendered by views/default/index.html or views/generic.html
+
+    if you need a simple wiki simply replace the two lines below with:
+    return auth.wiki()
+    """
+    response.flash = T("Hello World")
+    return dict(message=T('Welcome to web2py!'))
+
+
+@request.restful()
+def recieve_update():
```

Comment:
Can you add a doc comment below each method/procedure describing exactly what it does?